DOC-6271: Updated the Archive Service content (#1980)
corda/corda-docs-portal: content/en/tools/archiving-service/archiving-service-index.md

grant insert on all tables in schema corda to archive;
```

{{< note >}}

> You must execute the commands to create backup schema after the node has set up the main schema via its `run-migration-scripts` command.

Reviewer: ```suggestion
You must execute the commands to create backup schema after the node has set up the main schema using the `run-migration-scripts` command.
```

grant select, insert on corda.NODE_TRANSACTIONS to archive;
```

{{< note >}}
This applies to Oracle databases only; other database types don’t require table-by-table permissions updates.
{{< /note >}}

> The minimum list of tables that can/should be archived includes:

Reviewer: ```suggestion
The following tables should be archived:
```

* `ARCHIVABLE_TX`
* `ARCHIVABLE_ATT`
* `NODE_ATTACHMENTS`
* `NODE_TRANSACTIONS`

> There are also other archivable tables that contain data linked to `NODE_ATTACHMENTS` and `NODE_TRANSACTIONS`. An archivable table is one that contains either:

Reviewer: ```suggestion
There are also other archivable tables that contain data linked to `NODE_ATTACHMENTS` and `NODE_TRANSACTIONS`. An archivable table contains one of the following:
```

* A column named `transaction_id` (meaning it is linked to `NODE_TRANSACTIONS`).
* A column named `att_id` (meaning it is linked to `NODE_ATTACHMENTS`).

> Corda provides quite a few of these types of tables and they vary between different Corda versions. You can also add your own tables. Any tables that you want to include in the archiving process need to be granted access to the database as well.

Reviewer: ```suggestion
Corda provides multiple tables of this type and they vary between different Corda versions. You can also add your own tables. You must grant access to any tables that you want to include in the archiving process.
```
